### ISSUE TITLE
Armor is gone after retrieving grave with mod hardcore revival

### ISSUE THREAD

**Issue by firyy** [B1n-ry/Youre-in-grave-danger#91]
On death using hardcore revival, meaning **if you don't get revived in time**, when you go back to your grave you retrieve all your items except the armor.
It also seems that you can get revived after the timeout but you don't get "proper revived", this could be a problem on their end.

 - Minecraft Version: 1.19.2
 - YIGD Version: 1.4.6
 - Hardcore Revival: 9.0.1
 
Hope you can get a compat for this mod!

**Comment by B1n-ry** [comment]
Are you playing with twilight forest mod? Because that currently has a bug with the charm of keeping, voiding items instead of keeping them. So it could be your armor is voided because twilight thinks it should be kept (charm of keeping 1). I will try and look into the hardcore revival mod however, and see if there's anything I need to do to provide compat

**Comment by firyy** [comment]
No, not playing with any mod that adds that kind of items. I need to re-test this, think it only happens when reviving after the timeout which isn't supposed to happen. But still, only lose the armor, not the rest. Thanks :)

**Comment by B1n-ry** [comment]
I finally have proper time to work on this now. But I can't quite get the hardcore revival mod to work fully. Do I have to use it on a server for it to take effect? It didn't work when I tested on a LAN world (by working I mean it didn't say how long I could be rescued for and such. It was the normal death screen which I would assume the mod changes. It didn't look the way the image on the mod's curseforge page did anyway)

**Comment by B1n-ry** [comment]
I managed to get the mod working, but I could not reproduce the issue. Is this consistant for you?

**Comment by Landrozaum** [comment]
i recommend [Incapacitated Mod](https://www.curseforge.com/minecraft/mc-mods/incapacitated), since this mod never bugged out with YINGraveDanger and any other mod that adds gravestones, glad if i help.

Btw i think in general Incapacitated is better than Hardcore Revival since it is configurable and is compatible with some mods, and you can walk when lay down.

**Comment by B1n-ry** [comment]
Ok that's good to know. For anyone who would find a way to consitently reproduce the issue however I'd still like to resolve the issue with hardcore revival mod. Incapacitate mod might be a workaround but I still want to fix the issue
